sdk/core/azure-core/test/ut/datetime.cpp

```diff
@@ -594,3 +595,42 @@ TEST(DateTime, DefaultConstructible)
   DateTime dt;
   EXPECT_EQ(0, dt.time_since_epoch().count());
 }
+
+TEST(DateTime, ComparisonOperators)
```

Comment:
Please verify that this test will fail if we didn't have this change:
https://github.com/Azure/azure-sdk-for-cpp/pull/1326

Reply:
I did